.jenkins

```diff
@@ -5,7 +5,6 @@ pipeline {
   agent any
 
   environment {
-    PROJ_PATH = '/go/src/github.com/vapor-ware/sandbox-plugin'
     IMAGE_NAME = 'vaporio/sandbox-plugin'
     GOVERSION = '1.12'
```

Comment:
Did you want to clean up the GOVERSION env var here as well since its embedded in the builder image?

Reply:
yup - I was planning on doing this in the follow up "update CI to latest and greatest" PR. Wanted to keep the work for transitioning to go modules separate from greater CI updates.